create_package.sh

```diff
@@ -28,8 +28,22 @@ mv /ovms_release/lib/libcustom_node* /ovms_release/lib/custom_nodes/
 cd /ovms_release/lib/ ; rm -f libazurestorage.so.* ; ln -s libazurestorage.so libazurestorage.so.7 ;ln -s libazurestorage.so libazurestorage.so.7.5
 cd /ovms_release/lib/ ; rm -f libcpprest.so.2.10 ; ln -s libcpprest.so libcpprest.so.2.10
 if [ -f /ovms_release/lib/libopenvino_genai.so ]; then cd /ovms_release/lib/ ; rm -f libopenvino_genai.so.* ; ln -s libopenvino_genai.so libopenvino_genai.so.2510 ; ln -s libopenvino_genai.so.2025.1.0.0 libopenvino_genai.so.2510 ; fi
-if [ -f /ovms_release/lib/libopenvino_genai_c.so ]; then cd /ovms_release/lib/ ; rm -f libopenvino_genai_c.so.* ; ln -s libopenvino_genai_c.so libopenvino_genai_c.so.2510 ; ln -s libopenvino_genai_c.so.2025.1.0.0 libopenvino_genai_c.so.2510 ; fi
-rm -f /ovms_release/lib/libssl.so
+if [ -f /ovms_release/lib/libopenvino_genai_c.so ]; then cd /ovms_release/lib/ ; rm -f libopenvino_genai_c.so* ; fi
+
+# Remove GPU plugin for CPU images?
+# Remove OpenCL for CPU images?
+
+rm -rf \
+	/ovms_release/lib/libssl.so \
+	/ovms_release/lib/libexternal_Szlib_Slibzlib.so \
+	/ovms_release/lib/py_openvino_genai.cpython-312-x86_64-linux-gnu.so \
+	/ovms_release/lib/libinference_calculator_cc_proto.so \
+	/ovms_release/lib/libzlib.so \
+	/ovms_release/lib/libface_detection_cc_proto.so.so \
+	/ovms_release/lib/_crypt.cpython-312-x86_64-linux-gnu.so \
+	/ovms_release/lib/libface_detection_options_registry.so \
+	/ovms_release/lib/libinference_calculator_options_registry.so
```

Comment:
It would be worthy to check why those .so's exist in the first place

Reply:
yes, and I need to run full test suite

Reply:
tests passed